Language/de/Router.php

```diff
@@ -9,8 +9,10 @@
  * the LICENSE file that was distributed with this source code.
  */
 
-// Router language settings
 return [
-    'invalidParameter'    => 'Ein Parameter stimmt nicht mit dem erwarteten Typ überein.',
-    'missingDefaultRoute' => 'Es konnte nicht bestimmt werden, was angezeigt werden soll. In der Routingdatei wurde keine Standardroute angegeben.',
+    'invalidParameter'         => 'Ein Parameter stimmt nicht mit dem erwarteten Typ überein.',
+    'missingDefaultRoute'      => 'Es konnte nicht bestimmt werden, was angezeigt werden soll. In der Routingdatei wurde keine Standardroute angegeben.',
+    'invalidDynamicController' => 'Ein dynamischer Kontroller ist aus Sicherheitsgründen nicht erlaubt. Route handler: {0}',
+    'invalidControllerName'    => 'Das Namensraumbegrenzungszeichen ist ein Backslash (\), kein Schrägstrich (/). Route handler: {0}',
 ];
+// Router language settings
```

Comment:
Why was this moved?

Reply:
ui, that was not intentional. sorry. 
I will add the comment again.